### PR TITLE
vmui: add column search in table settings

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes

Add search functionality to the column display settings in the table
 #6668
 
image

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ The value of `instance` label for those scrape targets will be changed from `<ad
...
 * FEATURE: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation/): do not allow enabling `-stream.keepInput` and `keep_metric_names` options together in [stream aggregation config](https://docs.victoriametrics.com/stream-aggregation/#stream-aggregation-config), as it may result in time series collision.
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add search functionality to the column display settings in the table. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6668).
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add the ability to select all columns in the column display settings of the table. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6668).
```
Comment (hagen1778) [inline docs/CHANGELOG.md:47]: should this change be present in VictoriaLogs changelog as well?

File: app/vmui/packages/vmui/src/components/Table/TableSettings/TableSettings.tsx [hunk 2]
```diff
@@ -100,42 +128,64 @@ const TableSettings: FC<TableSettingsProps> = ({
...
                 />
+              )}
+              {!filteredColumns.length && (
+                <p className="vm-table-settings-popper-list__no-found">No columns found</p>
+              )}
+              <div className="vm-table-settings-popper-list-header">
+                {!!filteredColumns.length && (
+                  <div className="vm-table-settings-popper-list__item vm-table-settings-popper-list__item_check_all">
+                    <Checkbox
+                      checked={defaultColumns.length === columns.length}
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/Table/TableSettings/TableSettings.tsx:162]: Looks like this button should check or uncheck only filtered fields.
For example, if I filtered fields by search keyword and pressed check-all, I'd expect only the fields I see in the options to be checked. In fact, it checks all the fields.

**Review by hagen1778 — CHANGES_REQUESTED** [review]

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Pull request merged**